src/as-validator-issue-tag.h

```diff
@@ -204,6 +204,41 @@ AsValidatorIssueTag as_validator_issue_tag_list[] =  {
 	  N_("The update-contact does not appear to be a valid email address (escaping of `@` is only allowed as `_at_` or `_AT_`).")
 	},
 
+	{ "screenshot-invalid-width",
+	  AS_ISSUE_SEVERITY_ERROR,
+	  N_("The width property must be a integer")
+	},
+
+	{ "screenshot-invalid-height",
+	  AS_ISSUE_SEVERITY_ERROR,
+	  N_("The width property must be a integer")
+	},
+
+	{ "screenshot-image-invalid-type",
+	  AS_ISSUE_SEVERITY_ERROR,
+	  N_("The type of the image is neither source or thumbnail")
+	},
+
+	{ "screenshot-image-missing-width",
+	  AS_ISSUE_SEVERITY_ERROR,
```

Comment:
Should probably be a WARNING, the data isn't completely broken if this is missing (warnings will still make validation fail).